sentry/templates/deployment-sentry-post-process-forwarder.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: apps/v1
```

Comment:
Out of curiosity, what is this one supposed to do ? Fix the missing events ?

Reply:
Well I was going through sentry docker-compose and saw this one missing, so that was my guess on fix missing events :) not sure though